tests/baselines/reference/parseImportAttributesError.js

```diff
@@ -47,5 +47,5 @@ ImportInterface;
 
 //// [index.d.ts]
 export type LocalInterface = import("pkg", { with: {} });
-export declare const a: any;
-export declare const b: any;
+export declare const a: import("pkg", { with: {} });
+export declare const b: import("pkg", { with: {} });

```

Comment:
Are there any issues with preserving these attributes? I assume not, and that it's probably good to keep them.

Reply:
Now that they're not nightly-only, it should be fine to keep them around - you're more likely to randomly start breaking your consumers on older TS that you incidentally hadn't before, but... 🤷